siliconcompiler/core.py

```diff
@@ -1253,21 +1253,15 @@ def _find_files(self, *keypath, missing_ok=False, job=None, step=None, index=Non
             refdirs = self._find_files('tool', tool, 'task', task, 'refdir', step=step, index=index)
             search_paths = refdirs
 
-        import_steps = self._get_steps_by_task()['import']
         for path in paths:
             if (copyall or copy) and ('file' in paramtype) and not search_paths:
                 name = self._get_imported_filename(path)
-                found = False
-                for step, index in import_steps:
-                    abspath = os.path.join(self._getworkdir(jobname=job, step=step, index=index), 'inputs', name)
-                    if os.path.isfile(abspath):
-                        # if copy is True and file is found in import inputs,
-                        # continue. Otherwise, fall through to _find_sc_file (the
-                        # file may not have been gathered in imports yet)
-                        result.append(abspath)
-                        found = True
-                        break
-                if found:
+                abspath = os.path.join(self._getcollectdir(jobname=job), name)
+                if os.path.isfile(abspath):
+                    # if copy is True and file is found in import inputs,
+                    # continue. Otherwise, fall through to _find_sc_file (the
+                    # file may not have been gathered in imports yet)
```

Comment:
maybe update comment to say "collected inputs" or something along those lines instead of import